src/config.ts

```diff
@@ -99,6 +104,44 @@ class Config {
     return this.WALLET_SERVICE_BASE_URL;
   }
 
+  /**
+   * Returns the correct base url constant for the Atomic Swap Service.
+   * If the url was explicitly set using the config object, it is always returned.
+   * Otherwise, we return it based on the provided network parameter.
+   *
+   * If the url was not set in the config, and no network is provided, we throw an Error.
+   *
+   * @param network - The name of the network to be used to select the url.
+   * @return The Atomic Swap Service url
+   */
+  getSwapServiceBaseUrl(network?: 'mainnet'|'testnet'): string {
+    if (this.SWAP_SERVICE_BASE_URL) {
+      return this.SWAP_SERVICE_BASE_URL;
+    }
+
+    if (!network) {
+      throw new Error('You should either provide a network or call setSwapServiceBaseUrl before calling this.');
```

Comment:
:speech_balloon: Why not mension to environment variable alternative as well by configuring the `SWAP_SERVICE_BASE_URL` in the project?

Reply:
The headless and desktop wallets will certainly implement this as a environment variable, but I see that this should not be a `wallet-lib` recommendation for all clients.

Reply:
Mmmm... Well, I believe there is no problem to inform this alternative, after all this is an option the implementer has. If the client for some reason were not able to use an environment variable, this is a client's constraint, not the lib's constraint.